### PR TITLE
add possible values

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/office-docs-powershell/issues/10193

### CODE REVIEW

File: skype/skype-ps/skype/New-CsTeamsMobilityPolicy.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Accept wildcard characters: False
 ```
 
 ### -IPVideoMobileMode
-When set to WifiOnly, prohibits the user from making, receiving video calls or enabling video in meetings using VoIP calls on the mobile device while on cellular data connection.
+When set to WifiOnly, prohibits the user from making, receiving video calls or enabling video in meetings using VoIP calls on the mobile device while on cellular data connection. Possible values are: WifiOnly,AllNetworks.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMobilityPolicy.md:103]:
```suggestion
When set to WifiOnly, prohibits the user from making and receiving video calls or enabling video in meetings using VoIP calls on the mobile device while on a cellular data connection. Possible values are: WifiOnly, AllNetworks.

```

File: skype/skype-ps/skype/New-CsTeamsMobilityPolicy.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ Accept wildcard characters: False
 ```
 
 ### -IPAudioMobileMode
-When set to WifiOnly, prohibits the user from making, receiving calls or joining meetings using VoIP calls on the mobile device while on cellular data connection.
+When set to WifiOnly, prohibits the user from making, receiving calls or joining meetings using VoIP calls on the mobile device while on cellular data connection. Possible values are: WifiOnly, AllNetworks.
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMobilityPolicy.md:88]:
```suggestion
When set to WifiOnly, prohibits the user from making and receiving calls or joining meetings using VoIP calls on the mobile device while on a cellular data connection. Possible values are: WifiOnly, AllNetworks.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**